README.md

```diff
@@ -0,0 +1,86 @@
+# PyQtWebEngine BaseApp
+
+This base application is designed to be used for packaging a Flatpak application that depends on PyQt or PyQtWebEngine.
+
+**Note: This packaging is not affilated with, or supported by Riverbank Computing Limited.**
+
+## What's included?
+
+* [QtWebEngine base application](https://github.com/flathub/io.qt.qtwebengine.BaseApp)
+* [PyQt](https://riverbankcomputing.com/software/pyqt/) Python bindings for Qt
+* [PyQtWebEngine](https://riverbankcomputing.com/software/pyqtwebengine) Python bindings for QtWebEngine
+* Run-time depedencies for the above, including:
+  * freeglut
+  * glu
+  * krb5
+  * libevent
+  * python-opengl
```

Comment:
Where do these come from?

Reply:
krb5 & libevent are QtWebEngine dependencies.

I believe that python-opengl (freeglue+glu are dependencies) was needed in previous releases of PyQt5 if one was using the QtOpenGL bindings, but not anymore.  
It's still considered a dependency for PyQt5 in some distro, or at least in Arch Linux, maybe to avoid silently breaking HW acceleration in some applications that still use PyOpenGL.  
How many such apps are there? No idea. But having this in doesn't hurt, as the PyOpenGL interfaces should be pretty stable unlike other Python modules, and this can help developers working with PyQt and PyOpenGL, as they wouldn't need to build the binary dependencies of PyOpenGL.  
I can definitely drop this, but maybe keep in as a service for maintainers?

Reply:
I think `python-opengl` can be removed.  
From  
https://koji.fedoraproject.org/koji/rpminfo?rpmID=28220979  
https://packages.debian.org/en/source/bullseye/pyqt5  
fedora and debian both are not require `python-opengl` as build-depends(also runtime depends).  

pip(run in clean python venv) also not install `python-opengl`:
```
> pip install pyqt5
Collecting pyqt5
  Using cached PyQt5-5.15.6-cp36-abi3-manylinux1_x86_64.whl (8.3 MB)
Collecting PyQt5-Qt5>=5.15.2
  Using cached PyQt5_Qt5-5.15.2-py3-none-manylinux2014_x86_64.whl (59.9 MB)
Collecting PyQt5-sip<13,>=12.8
  Downloading PyQt5_sip-12.10.1-cp310-cp310-manylinux1_x86_64.whl (352 kB)
     |████████████████████████████████| 352 kB 387 kB/s 
Installing collected packages: PyQt5-sip, PyQt5-Qt5, pyqt5
Successfully installed PyQt5-Qt5-5.15.2 PyQt5-sip-12.10.1 pyqt5-5.15.6
```

If a application really need `python-opengl`, it should manually include it.

Reply:
I only checked Arch Linux and Fedora. If it's also PyPI and Debian, then let's drop them.

Reply:
Resolved in 852ee02abdf6790e8ea52769975fbd208d1ae847.